source/community/reactnative/experimental/adaptive-card-builder/AdaptiveCardBuilder.js

```diff
@@ -0,0 +1,196 @@
+
+import * as Constants from './constants';
+
+/**
+ * @description - This builder creates adaptive card from `HeroCard` / `ThumbnailCard` .
+ * @param  cardContent  - Content of the card to be converted
+ * @param  type  - Type of the card.
+ */
+export const buildAdaptiveCard = (cardContent, type) => {
+
+    const AdaptiveCard = {
+        "type": Constants.TypeAdaptiveCard,
+        "version": Constants.AdaptiveCardVersion,
+        "body": [],
+        "actions": []
+    }
+
+    const ColumnSet = {
+        "type": Constants.TypeColumnSet,
+        "columns": [
+            {
+                "type": Constants.TypeColumn,
+                "width": Constants.ThumbNailWidth,
+                "items": []
+            },
+            {
+                "type": Constants.TypeColumn,
+                "width": Constants.WidthStretch,
+                "items": []
+            }
+        ]
+    }
+
+    if (cardContent && type) {
```

Comment:
Always clone JSON templates instead of manipulating on the object directly. This will help in restoring the const template if required. 

function clone(object) {
  return (JSON.parse(JSON.stringify(object)));
}

Reply:
As discussed, If the template is not going to be used later, This looks fine to me